config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -3,32 +3,44 @@ presubmits:
   - name: pull-kubernetes-e2e-gci-gce-ipvs
     always_run: false
     optional: true
-    skip_report: true
+    skip_report: false
```

Comment:
I initially suggested to skip_report, but I found that since it's optional it won't run until it is called, 
making it difficult to debug because you can't see the output from the PR you are calling it 
my fault

Reply:
trying to parse this ... " since the job optional, it won't run until it is called,"

`always_run` and `run_if_changed` control if it runs.
`optional` is a way to make something that runs not required for merge
`skip_report` is a way to make it not post a comment or status on the PR. you can still see the results in prow.k8s.io / testgrid

Reply:
hehe, I misunderstood the `optional` thing :man_shrugging: 

Let me rephrase since this job can only run manually and is optional, there is no risk that somebody runs it accidentally and/or block any PR, thus reporting the status if it runs should not be a problem

Reply:
> Let me rephrase: since this job can only run manually and is optional, there is no risk that somebody runs it accidentally and/or block any PR, thus reporting the status when runs should not be a problem, because it means that , who runs the job, wants to see the result sweat_smile

Make sense to me and this was my original intent, thanks for fixing @aojea!